python/cuspatial/CMakeLists.txt

```diff
@@ -44,12 +44,8 @@ if(NOT cuspatial_FOUND)
 
   set(cython_lib_dir cuspatial)
   include(cmake/Modules/WheelHelpers.cmake)
-  # TODO: This install is currently overzealous. We should only install the libraries that are
-  # downloaded by CPM during the build, not libraries that were found on the system.  However, in
-  # practice this would only be a problem if libcudf was not found but some of the
-  # dependencies were, and we have no real use cases where that happens.
   install_aliased_imported_targets(
-    TARGETS cuspatial arrow_shared nvcomp::nvcomp nvcomp::nvcomp_gdeflate nvcomp::nvcomp_bitcomp
+    TARGETS cuspatial
```

Comment:
Actually, now that we've removed all of these targets we should just remove the WheelHelpers altogether and just do a normal CMake `install`.

Reply:
Great point! Done in https://github.com/rapidsai/cuspatial/pull/1447/commits/fe7df5d7c0d55b085be4ae9e2d23de31df2fc169